OptimizelySDK/Odp/OdpSegmentApiManager.cs

```diff
@@ -231,28 +253,19 @@ string query
                         Constants.HEADER_API_KEY, apiKey
                     },
                 },
-                Content = new StringContent(query, Encoding.UTF8, "application/json"),
+                Content = new StringContent(query, Encoding.UTF8,
+                    Constants.APPLICATION_JSON_MEDIA_TYPE),
             };
 
             return request;
         }
 
-        /// <summary>
-        /// Ensure a string has content that can be parsed from JSON to an object
-        /// </summary>
-        /// <param name="jsonToValidate">Value containing possible JSON</param>
-        /// <returns>True if content could be interpreted as JSON else False</returns>
-        private static bool CanBeJsonParsed(string jsonToValidate)
-        {
-            return string.IsNullOrWhiteSpace(jsonToValidate);
-        }
-
         /// <summary>
         /// Parses JSON response
         /// </summary>
         /// <param name="jsonResponse">JSON response from ODP</param>
         /// <returns>Strongly-typed ODP Response object</returns>
-        public static Response ParseSegmentsResponseJson(string jsonResponse)
+        public static Response DeserializeSegmentsFromJson(string jsonResponse)
```

Comment:
Can you please add a test case if not written, in providing invalid json, what will be the behavior?